src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -231,6 +237,45 @@ private async Task RunLoop(CancellationToken cancellationToken)
                         issue.AugmentedIndex = "999." + issue.Index.ToString("D3");
                     }
                 }
+            }      
+
+            TimeSpan cancelationTime; 
+            if(!TimeSpan.TryParseExact(_options.Value.LogScrapingTimeout, @"m\:s\:fff", CultureInfo.CurrentCulture, out cancelationTime)){
```

Comment:
This is not ideal. If someone enters an invalid string in the settings, we want to know and blow up here so they know their timeout is not being used.

I'd just do:
```csharp
var cancelationTime = TimeSpan.Parse(_options.Value.LogScrapingTimeout ?? "00:10:00");
```

but not sure whether there can be empty string there, you need to check ^^

Reply:
I would make it explode, really

Reply:
I made it explode